packages/config-vite-base/package.json

```diff
@@ -17,5 +17,12 @@
   "devDependencies": {
     "typescript": "^5.8.3"
   },
-  "private": true
+  "private": true,
+  "exports": {
```

Comment:
Why are these changes needed?

Reply:
Were you running into some problems? Since this change is not related to the issue at hand. Totally okay to keep it in, just curious where it was affecting you